ddtrace/tracer/option.go

```diff
@@ -168,6 +168,19 @@ func (c *config) HasFeature(f string) bool {
 	return ok
 }
 
+func (c *config) DumpForTestAgent() string {
```

Comment:
This must be private. Also conceptually there is no reason why the internals depend on the implementation details of the tests. Can we find another way to pass this information instead of having this custom logic here?

Reply:
removed